### ISSUE TITLE
Legend textDecoration doesn't work in IE11 with accessibility module.

### ISSUE THREAD

**Issue by raf18seb** [highcharts/highcharts#12161]
#### Actual behaviour
```
legend: {
    itemStyle: {
      textDecoration: null
    },
    itemHiddenStyle: {
      textDecoration: 'line-through'
    }
  },
```
textDecoration property doesn't work when accessibility module is attached (only in IE11).
Works fine on different browsers or when we remove the accessibility module. 

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/vkgxLfuj

#### Product version
Highcharts 7.2.0

#### Affected browser(s)
Only IE11 (tested on Windows 10).

@oysteinmoseng
